plugins/pulp_rpm/plugins/importers/yum/associate.py

```diff
@@ -52,6 +52,12 @@ def associate(source_repo, dest_repo, import_conduit, config, units=None):
     if recursive is None:
         recursive = False
 
+    if recursive:
+        solver = pulp_solv.Solver(source_repo, target_repo=dest_repo)
+        solver.load()
+    else:
+        solver = None
```

Comment:
what about getting rid of unnecessary if statement on L#52 and:
```
    if recursive:
        solver = pulp_solv.Solver(source_repo, target_repo=dest_repo)
        solver.load()
    else:
        solver = None
        recursive = False
```
in my understanding this option is a boolean which is whether true or false.
in addition i noticed that this option is not documented in technical reference at all https://docs.pulpproject.org/plugins/pulp_rpm/tech-reference/yum-plugins.html#yum-importer

Reply:
i'd also set a default value to False:
```
# get config items that we care about
recursive = config.get(constants.CONFIG_RECURSIVE, False)
```

Reply:
lemme update

Reply:
if you could also update the tech. reference docs would be great

Reply:
Sure I'll check it